src/Api/Helpers/AddDatabaseExtensionMethod.cs

```diff
@@ -29,6 +27,7 @@ public static IServiceCollection AddDatabase(this IServiceCollection services, I
                 // resolving config from SP to avoid capturing
                 builder.UseSqlite(sp.GetRequiredService<IConfiguration>().GetConnectionString("sqlite:api"));
             });
+            services.AddScoped<ITenantStorage, EfTenantStorage>();
```

Comment:
nitpitck, duplicated: We can de-duplicate this row by moving it of the IF, e.g. together with ITenantProvider.

Reply:
@abergs done